src/MusicalScore/ScoreIO/MusicSymbolModules/LyricsReader.ts

```diff
@@ -71,28 +71,9 @@ export class LyricsReader {
                                     syllabic = "middle";
                                 }
                             }
-                            let currentLyricVerseNumber: number = 1;
-                            let errorNumberParse1: boolean = false;
+                            let currentLyricVerseNumber: string = "1";
                             if (lyricNode.attributes() !== undefined && lyricNode.attribute("number")) {
-                                try {
-                                    currentLyricVerseNumber = parseInt(lyricNode.attribute("number").value, 10); // usually doesn't throw error, but returns NaN
-                                } catch (err) {
-                                    errorNumberParse1 = true;
-                                }
-                                errorNumberParse1 = errorNumberParse1 || isNaN(currentLyricVerseNumber);
-                                if (errorNumberParse1) {
-                                    try { // Sibelius format: "part1verse1"
```

Comment:
Nice that you could simplify the code and delete the Sibelius format fix without breaking it (originally I expected it to be broken).
Though I'm not sure that having the verse number as string instead of number is always a win.
Of course it is when the xml gives verse numbers as random strings instead of numbers, like Sibelius (which is bad at XML export).

Reply:
Oh, right, if this is what the MusicXML specification says, we unfortunately need to stick with it ;) thanks for the info!